scripts/r.fillnulls/testsuite/test_r_fillnulls.py

```diff
@@ -55,7 +54,16 @@ def test_basic(self):
             overwrite=True,
         )
         self.assertModule(module)
-        self.assertRasterFitsUnivar(raster=self.mapComplete, reference=self.values)
+        self.assertRasterFitsUnivar(
+            raster=self.mapComplete,
+            reference={
+                "null_cells": float(0),
+                "max": 130.913299560547,
+                "range": 73.6245880126953,
+                "variance": 288.817309946793,
+            },
+            precision=1e-6,
```

Comment:
This is precision is fine, but then you don't need so many decimal places in the reference values.